package.json

```diff
@@ -73,7 +77,7 @@
         },
         {
           "value": "nl",
-          "description": "Nederlands (nl)"
+          "description": "Nederlandhttps://help.github.com/articles/syncing-a-fork/https://help.github.com/articles/syncing-a-fork/https://help.github.com/articles/syncing-a-fork/https://help.github.com/articles/syncing-a-fork/s (nl)"
```

Comment:
some reminder: 

after you make changes, please push those new commits to 
`francematt:master` and this page will automatically loa latest code.